.github/actions/external-repository-setup/action.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Setup External Repository w/ Branch Setup'
```

Comment:
Delete this file. It is a subset of https://github.com/nasa/fprime-actions/blob/devel/external-repository-setup/action.yml